### DISCUSSION TITLE
Does Job Runner Support Updating MongoDB Connection if it gets timed out since token is not valid

### DISCUSSION THREAD

**Discussion by nagabhushanss** [jobrunr/jobrunr#574] [Q&A]
We are connecting to Mongo using AWS role/sts mechanism. We are connecting jobRunner to this Mongodb connection during service startup. This connection does get timed out after every one hour. We are able to handle at the component level but we also want to update the connection information that we would have registered with job runner during starting of service. Does jobrunner expose method to refresh the underlying Storage Configuration mechanism ?

**Comment by rdehuyss** [comment]
Hi @nagabhushanss , I'm afraid the connection to MongoDB is currently only supporting username and password authentication.
